tfhe/docs/getting_started/benchmarks.md

```diff
@@ -1,44 +1,44 @@
 
 ## Booleans
 
 This measures the execution time of a single binary boolean gate.
 
-### thfe.rs::booleans
+### thfe.rs::booleans.
 
-| Parameter set | concrete-fft | concrete-fft + avx512 |
-| --- | --- | --- |
-| DEFAULT_PARAMETERS | 8.8ms | 6.8ms |
-| TFHE_LIB_PARAMETERS | 13.6ms | 10.9ms |
+| Parameter set         | concrete-fft | concrete-fft + avx512 |
+| --------------------- | ------------ | --------------------- |
+| DEFAULT\_PARAMETERS   | 8.8ms        | 6.8ms                 |
+| TFHE\_LIB\_PARAMETERS | 13.6ms       | 10.9ms                |
 
-### tfhe-lib
+### tfhe-lib.
 
-| Parameter set  | fftw | spqlios-fma|
-| --- | --- | --- |
-| default_128bit_gate_bootstrapping_parameters | 28.9ms | 15.7ms |
+| Parameter set                                    | fftw   | spqlios-fma |
+| ------------------------------------------------ | ------ | ----------- |
+| default\_128bit\_gate\_bootstrapping\_parameters | 28.9ms | 15.7ms      |
 
-### OpenFHE
+### OpenFHE.
 
-| Parameter set  | GINX | GINX (Intel HEXL) |
-| --- | --- | --- |
-| STD_128 | 172ms | 78ms |
-| MEDIUM | 113ms | 50.2ms |
+| Parameter set | GINX  | GINX (Intel HEXL) |
+| ------------- | ----- | ----------------- |
+| STD\_128      | 172ms | 78ms              |
+| MEDIUM        | 113ms | 50.2ms            |
 
-## Shortints
-This measures the execution time for some operations and some parameter sets of shortints.
+## Shortint
 
-### thfe.rs::shortint
-This uses the concrete-fft + avx512 configuration.
+This measures the execution time for some operations and some parameter sets of shortint.
+
+### thfe.rs::shortint.
 
+This uses the concrete-fft + avx512 configuration.
 
-| Parameter set                    | unchecked_add  |  unchecked_mul_lsb       |  keyswitch_programmable_bootstrap     |
-| ---                     | ---            | ---                      | ---                      |
-| PARAM_MESSAGE_1_CARRY_1 | 337 ns         | 10.1 ms                  | 9.91 ms                   |
-| PARAM_MESSAGE_2_CARRY_2 | 407 ns         | 21.7 ms                  | 21.4 ms                    |
-| PARAM_MESSAGE_3_CARRY_3 | 3.06 µs        | 161  ms                  | 159 ms                     |
-| PARAM_MESSAGE_4_CARRY_4 | 11.7 µs        | 1.03 s                   | 956 ms                     |
+| Parameter set               | unchecked\_add | unchecked\_mul\_lsb | keyswitch\_programmable\_bootstrap |
+| --------------------------- | -------------- | ------------------- | ---------------------------------- |
+| PARAM\_MESSAGE\_1\_CARRY\_1 | 337 ns         | 10.1 ms             | 9.91 ms                            |
+| PARAM\_MESSAGE\_2\_CARRY\_2 | 407 ns         | 21.7 ms             | 21.4 ms                            |
+| PARAM\_MESSAGE\_3\_CARRY\_3 | 3.06 µs        | 161 ms              | 159 ms                             |
+| PARAM\_MESSAGE\_4\_CARRY\_4 | 11.7 µs        | 1.03 s              | 956 ms                             |
```

Comment:
we checked yesterday with Ilaria, those numbers are actually incorrect (the actual values were in the files but some older results were duplicated)

```
| Parameter set               | unchecked\_add | unchecked\_mul\_lsb | keyswitch\_programmable\_bootstrap |
| --------------------------- | -------------- | ------------------- | ---------------------------------- |
| PARAM\_MESSAGE\_1\_CARRY\_1 | 338 ns         | 8.3 ms             | 8.1 ms                            |
| PARAM\_MESSAGE\_2\_CARRY\_2 | 406 ns         | 18.4 ms             | 18.4 ms                            |
| PARAM\_MESSAGE\_3\_CARRY\_3 | 3.06 µs        | 134 ms              | 134 ms                             |
| PARAM\_MESSAGE\_4\_CARRY\_4 | 11.7 µs        | 854 ms            | 945 ms                             |
```

Reply:
done locally